public/app/features/canvas/runtime/scene.tsx

```diff
@@ -375,10 +411,88 @@ export class Scene {
     });
   };
 
+  contextMenuAction = (actionType: string) => {
+    this.selection.pipe(first()).subscribe((currentSelectedElements) => {
+      const currentSelectedElement = currentSelectedElements[0];
+      const currentLayer = currentSelectedElement.parent!;
+
+      switch (actionType) {
+        case LayerActionID.Delete:
+          currentLayer.doAction(LayerActionID.Delete, currentSelectedElement);
+          break;
+        case LayerActionID.Duplicate:
+          currentLayer.doAction(LayerActionID.Duplicate, currentSelectedElement);
+          break;
+        case LayerActionID.MoveTop:
+          currentLayer.doAction(LayerActionID.MoveTop, currentSelectedElement);
+          break;
+        case LayerActionID.MoveBottom:
+          currentLayer.doAction(LayerActionID.MoveBottom, currentSelectedElement);
+          break;
+      }
+    });
+  };
+
+  renderMenuItems = () => {
+    return (
+      <>
+        <MenuItem
+          label="Delete"
+          onClick={(e) => {
+            this.contextMenuAction(LayerActionID.Delete);
+            this.closeContextMenu();
+          }}
+          className={this.styles.menuItem}
+        />
```

Comment:
It seems like the `Delete` option is always highlighted when opening the context menu and selecting other actions / regardless if `Delete` option is being hovered or not
<img width="310" alt="Delete option always stays highlighted" src="https://user-images.githubusercontent.com/22381771/168714595-11abcdd7-0ff6-476f-bf56-d3a7cf7cc18a.png">